jetty-util/src/main/java/org/eclipse/jetty/util/Pool.java

```diff
@@ -212,7 +238,7 @@ public final void setMaxMultiplex(int maxMultiplex)
     @ManagedAttribute("The default maximum usage count of entries")
     public int getMaxUsageCount()
     {
-        return maxUsageCount;
+        return maxUsage;
```

Comment:
It's already been concluded that max usage should not have been implemented in the low-level `Pool` but rather in the higher level `ConnectionPool`, like what we did for the max-time-to-live. If you're considering deprecating the max multiplex, this other property should be deprecated too.